src/System.Data.Odbc/tests/System.Data.Odbc.Tests.csproj

```diff
@@ -34,6 +34,16 @@
       <Link>Common\Interop\OSX\Interop.Libraries.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetsFreeBSD)' == 'true'">
+    <Compile Include="$(CommonPath)\Interop\FreeBSD\Interop.Libraries.cs">
+      <Link>Common\Interop\FreeBSD\Interop.Libraries.cs</Link>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetsUnknownUnix)' == 'true'">
+    <Compile Include="$(CommonPath)\Interop\UnknownUnix\Interop.Libraries.cs">
+      <Link>Common\Interop\FreeBSD\Interop.Libraries.cs</Link>
```

Comment:
This should be \Interop\UnknownUnix\, right?

Reply:
It could. We can fork it later if the values become different. Note that Linux currently has same value as well. e.g. "libodbc.so.2"

Reply:
I'm not understanding.  You're Include'ing UnknownUnix:
```
<Compile Include="$(CommonPath)\Interop\UnknownUnix\Interop.Libraries.cs">
```
but calling it FreeBSD:
```
<Link>Common\Interop\FreeBSD\Interop.Libraries.cs</Link>
```